packages/react-components/lib/item-unknown.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core';
+import ErrorIcon from '@material-ui/icons/Error';
+import { Typography, Grid } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  errorIcon: {
+    color: theme.palette.error.main,
+  },
+  errorMsg: {
+    margin: '0 0.5rem',
+  },
+  errorDisabled: {
+    pointerEvents: 'none',
+    filter: 'blur(.25rem)',
+  },
+}));
+
+export interface ItemUnknownProps {
+  errorMsg?: string;
+  showError: boolean;
+  children: JSX.Element | null;
+}
+
+export const ItemUnknown = (props: ItemUnknownProps): JSX.Element => {
+  const classes = useStyles();
+  const { errorMsg, showError, children } = props;
+
+  return showError ? (
+    <React.Fragment>
+      <Grid container direction="row" justify="center" alignItems="center" spacing={2}>
+        <Grid item>
+          <Typography color="error" variant="h6" align="center">
+            Error
+          </Typography>
+        </Grid>
+        <Grid item>
+          <ErrorIcon className={classes.errorIcon} />
+        </Grid>
+      </Grid>
+      <Typography className={classes.errorMsg} color="error" variant="body1" align="center">
```

Comment:
I don't think it's needed now but we could look at adding a prop/props to let the consumer to style the error message.